README.md

```diff
@@ -1,31 +1,40 @@
-# Galvanize Q3 Group Project Proposal
+# Galvanize Group Project Proposal
 
-1. Fork/clone this repo
-2. Update this readme with proposal
-  * Use valid markdown formatting!
-3. Pull request
+* 1 member from your team will need to fork this repo and update this README file with your proposal.
+* Make sure to preview your proposal in a markdown preview and [use valid markdown syntax](https://help.github.com/articles/basic-writing-and-formatting-syntax/)
+  * Unformatted/unreadable proposals will be rejected
+
+## Team Name
+### Musically driven
 
 ## Group Members
+Lena, Michelle, Molly, Taylor, Nick F., Jordan D.
 
+## Project/Application Name
+### Concert Spot
 
 ## Project Description
-
+A mobile app that gathers local concert information and creates a Spotify playlist with those artists. Vote up or down to see concerts you should check out!
 
 ## Who uses it?
-
+Everyone interested in live music!
 
 ## What outputs do they need?
-
+- local concerts
+- preferred music
+- playlist
 
 ## What inputs are needed to generate those outputs?
-
-
-## What technologies will you use that weren't covered in class?
-
+- location
+- upvote or downvote songs
 
 ## What technologies do you plan to use?
-
+- React Native
+- Meteor?
+- Spotify API
+- Concert API (songkick)
 
 ## Feature list
-
-
+- Playlist based on location
+- Voting system
+- Save concerts based on upvoted songs
```

Comment:
I have a feeling your feature list will grow as you work on this, but this is a great place to start